### ISSUE TITLE
example-app generates errors on install

### ISSUE THREAD

**Issue by paulbhartzog-ps** [simplabs/ember-simple-auth#1453]
The default instructions that are listed at:

https://github.com/simplabs/ember-simple-auth#example-app

generates the following error:

> ember-cli-htmlbars-inline-precompile v1.0.0 and above require the ember-cli-babel v6.7.1 or above. To use ember-cli-babel v5.x please downgrade ember-cli-htmlbars-inline-precompile to v0.3. An error occurred in the constructor for ember-cli-htmlbars-inline-precompile at /ember-simple-auth/node_modules/ember-cli-htmlbars-inline-precompile

The "issue" I'm reporting isn't the actual error, which may be simple to fix.  The issue is that your website documentation should have accurate instructions for all the newbies who want to fire up the Example App. ;-)

Much thx.

**Comment by marcoow** [comment]
@Turbo87: I'm not sure how the instructions

```
git clone https://github.com/simplabs/ember-simple-auth.git
cd ember-simple-auth
yarn install && ember serve
```

could ever lead to that error as we should see that on Travis as well then.

**Comment by paulbhartzog-ps** [comment]
Let me know what you learn. I didn't make any changes.  I'd like to know if it works on your end in case it might have been something local to my environment.  Much thx.

**Comment by Turbo87** [comment]
> could ever lead to that error as we should see that on Travis as well then.

Since those instructions are using `yarn install` the `yarn.lock` file will be taken into account, which it currently installing `ember-cli-babel@6.6.0` for this package. For TravisCI we are using `yarn install --no-lockfile` to make sure to test against the latest possible dependencies that a user would also install when doing `yarn add ember-simple-auth`.

tl;dr we should `yarn upgrade -L ember-cli-babel` and then the issue should be solved

**Comment by marcoow** [comment]
> Since those instructions are using yarn install the yarn.lock file will be taken into account, which it currently installing ember-cli-babel@6.6.0 for this package.

But that means the `yarn.lock` file is in a bad state actually, right?

**Comment by Turbo87** [comment]
> But that means the yarn.lock file is in a bad state actually, right?

no, it just means that it does its job as a lockfile

**Comment by marcoow** [comment]
Yes, but the content of the `yarn.lock` file is bad as in when you install exactly the versions referenced in it, things will not work. We don't see that on Travis as we're not using the lock file there. I think it would probably make sense to execute the Travis build with both `yarn install` and `yarn install --no-lockfile` so we see errors like this early.

**Comment by Turbo87** [comment]
> Yes, but the content of the yarn.lock file is bad as in when you install exactly the versions referenced in it, things will not work.

it works, now that #1459 is merged

> I think it would probably make sense to execute the Travis build with both yarn install and yarn install --no-lockfile so we see errors like this early.

if you want to double the CI runtime for a IMHO very marginal benefit...

**Comment by marcoow** [comment]
Let's see whether something like this comes up again. I do think it is a problem that we don't have anything in place that ensures the contents of `yarn.lock` actually work, especially as we're using Greenkeeper and thus commit dependency updates that never run on anyone's dev machine.
